client/my-sites/subscribers/strings-for-translation.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unused-vars */
+
+// This is a temporary file that introduces strings that need to be translated.
+// It will be removed once the page is complete and the strings are translated.
+
+import { translate } from 'i18n-calypso';
+
+// username is needed for some of the strings below
+const username = 'username_placeholder';
+
+const importSubscribers = translate( 'Import subscribers' );
+const allSubscribers = translate( 'All subscribers' );
+
+const subscribers = translate( 'Subscribers' );
+const downloadCSV = translate( 'Download CSV' );
+const addSubscribers = translate( 'Add Subscribers' );
+
+const search = translate( 'Search by name, username or email…' );
+const bulkActions = translate( 'Bulk actions' );
+
+const name = translate( 'Name' );
+const status = translate( 'Status' );
+const active = translate( 'Active' );
+const inactive = translate( 'Inactive' );
+const openRate = translate( 'Open rate' );
+const subscriptionType = translate( 'Subscription type' );
+const free = translate( 'Free' );
+const monthly = translate( 'Monthly' );
+const yearly = translate( 'Yearly' );
+const all = translate( 'All' );
+const subscriptionDate = translate( 'Subscription date' );
+const since = translate( 'Since' );
+const view = translate( 'View' );
+const passwordReset = translate( 'Send password reset' );
+const remove = translate( 'Remove' );
```

Comment:
Great catch! I left this in on purpose in case we change our mind with the design. Anyways, both "Unsubscribe" and "Remove" strings are prevalent in the codebase - so they have been already translated. I added them to this PR just for completeness.